docs/_news/news09.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: news
+title: New for Codewind 0.9.0
+description: New for Codewind 0.9.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news09
+---
+
+## Codewind 0.9.0
+Tuesday, 18 February 2020
+
+#### ✨ New Features and Highlights for 0.9.0 ✨
+Introducing the tech preview of Codewind on the IntelliJ IDE! We want to make our tools available to as many developers as possible in their preferred IDEs. Many people have asked about IntelliJ, as it is a popular option among Java developers, so we are working hard to bring the effortless Codewind experience to IntelliJ. With this tech preview, you can now use Codewind on IntelliJ locally.
+![](images/imagesfornews/intelliJ-techpreview.gif){:width="800"}
+
+**IDEs (Both Eclipse and VS Code)**
+- Detailed application status notifications provide a [link to documentation if a problem is encountered](https://github.com/eclipse/codewind/issues/1812).
+- Depending on the level of support, you can [change the logging level in the IDE](https://github.com/eclipse/codewind/issues/1251). In Eclipse, you can enable support features and set the loggin level in Codewind preferences. In VS Code, you can access this configuration in the command palette - `Set Codewind Server Logging Level`.
+
+**VS Code**
+- If a remote connection disconnects due to network changes, such as the local machine going into sleep mode or switching networks, you can simply [refresh to reconnect](https://github.com/eclipse/codewind/issues/1776). 
+
+**Eclipse**
+- When using Codewind remotely in Eclipse, you can [create a new project and select the remote connection](https://github.com/eclipse/codewind/issues/2014). You can also [choose to add a project to a specific remote connection if there is more than one connection](https://github.com/eclipse/codewind/issues/1695). 
+- The Add Image Registry dialog includes a [drop down of common registry addresses](https://github.com/eclipse/codewind/issues/1907). 
+
+#### List of Fixes
+- The build application queue [supports applications that are being updated](https://github.com/eclipse/codewind/issues/1563), not just project creation or restart.
+- You can [define references to files that reside outside of a project](https://github.com/eclipse/codewind/issues/1399) by creating the .cw-refpaths.json file in your project to specify the path reference.
+- [`Inject Metrics` is supported for Generic Docker Java projects](https://github.com/eclipse/codewind-vscode/pull/425). Previously, the action appeared on all of these projects, but [only worked on Open Liberty](https://github.com/eclipse/codewind/issues/1509).
```

Comment:
This is not correct, it still only works for Open Liberty. The fact that `inject metrics` is shown for all docker java projects is a bug https://github.com/eclipse/codewind/issues/2223

Reply:
I know this is copied from my slack message so that's my bad for not communicating it is explicitly NOT supported EXCEPT for open liberty :)

Reply:
ohhh, okay. i got really confused, because in 0.7.0 we talked about how we could inject metrics into node, spring, and microprofile projects already (https://www.eclipse.org/codewind/news07.html). 

I'll correct 0.9.0 to say that it's only supported oh open liberty

Reply:
This one is a little tricky to write/place. It's a little weird to have "works on OL, but  by the way, there's a bug that hasn't been fixed" :/

Just to verify --- the bug has not been fixed, right?

- [`Inject Metrics` is supported for Generic Docker Java projects](https://github.com/eclipse/codewind-vscode/pull/425). 

- The action appears on all projects, but [only works on Open Liberty](https://github.com/eclipse/codewind/issues/1509).

Reply:
@tobespc what do you think

Reply:
it is a bug for lagom but we can only enable metrics for liberty or open liberty.  Its not generic to all java projects

Reply:
so is this okay to go into the update:
- Inject Metrics is supported for [Liberty](https://github.com/eclipse/codewind-vscode/pull/425) and [Open Liberty](https://github.com/eclipse/codewind/issues/1509) projects.